packages/blocks/src/api/parser/convert-legacy-block.js

```diff
@@ -67,5 +72,8 @@ export function convertLegacyBlockNameAndAttributes( name, attributes ) {
 		name = 'core/comment-date';
 	}
 
-	return [ name, newAttributes ];
+	return applyFilters( 'editor.convertLegacyBlockNameAndAttributes', [
```

Comment:
Can we document the filter in the code with JSDoc? For reference, this is how filters get documented in WordPress core:

https://github.com/WordPress/wordpress-develop/blob/e94cd298c6e0208a5feb888099601feadf460eda/src/wp-includes/blocks.php#L262-L269

We should also include a new section in this document:

https://github.com/WordPress/gutenberg/blob/trunk/docs/reference-guides/filters/block-filters.md#block-editor

Reply:
That would be great. Cc @ryanwelcher.